### PR TITLE
Adding BETTER Mobile Threat Defense data connector, sample workbook and sample data

### PR DESCRIPTION
**Pull request by aymenim**
## Proposed Changes

  - Add BETTER MTD data connector
  - Add BETTER MTD workbook
  - Add BETTER MTD logo
  - Add BETTER MTD sample data

### CODE REVIEW

File: DataConnectors/BETTER MTD.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            "legend": "BetterMTDIncidentLog_CL",
+            "baseQuery": "BetterMTDIncidentLog_CL"
+        },
+        {
+            "metricName": "Total devcie enrolled",
+            "legend": "BetterMTDDeviceLog_CL",
+            "baseQuery": "BetterMTDDeviceLog_CL"
+        },
+        {
+            "metricName": "Total netflpw traffic recieved",
```
Comment (nazang) [inline DataConnectors/BETTER MTD.json:18]: typo? netflow

File: Sample Data/Custom/BetterMTDDeviceLog_CL.csv [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿TenantId,SourceSystem,TimeGenerated,RawData,"BuildNumber_s","Manufacturer_s","DeviceId_d","DeviceUDID_g","DeviceOS_s","CompanyId_d","DevicePlatform_s","LastReported_t","LocationID_d","CompanyName_s","UserEmail_s","IsDeleted_d","ThreatLevel_s","ThreatScore_d","AddedDate_t","AgentVersion_s",Type
```
Comment (nazang) [inline Sample Data/Custom/BetterMTDDeviceLog_CL.csv:1]: @preetikr, is there a requirement to have sample data in .json file?

**Review by nazang — CHANGES_REQUESTED** [review]

File: DataConnectors/BETTER MTD.json (hunk 1 above)
Comment (aymenim) [reply DataConnectors/BETTER MTD.json]: Yes, thanks for pointing that out, I have fixed that.

File: Sample Data/Custom/BetterMTDDeviceLog_CL.csv (hunk 2 above)
Comment (nazang) [reply Sample Data/Custom/BetterMTDDeviceLog_CL.csv:1]: @aymenim, please provide json format sample data. According to the guideline, Json format sample is required for rest API connecotr.

File: DataConnectors/BETTER MTD.json [hunk 3]
```diff
@@ -0,0 +1,126 @@
+{
```
Comment (nazang) [inline DataConnectors/BETTER MTD.json:1]: No space in file name please. 
Rename the connector file name to BETTERMTD.json

File: Workbooks/BETTER_MTD_Workbook.json [hunk 4]
```diff
@@ -0,0 +1,718 @@
+{
```
Comment (nazang) [inline Workbooks/BETTER_MTD_Workbook.json:1]: Need to update the Workbook metadata file WorkbooksMetadata.json  to include the new workbook. This is critical and add workbook metadata definition.

**Review by nazang — CHANGES_REQUESTED** [review]

File: DataConnectors/BETTER MTD.json (hunk 3 above)
Comment (aymenim) [reply DataConnectors/BETTER MTD.json]: Renamed

File: Workbooks/BETTER_MTD_Workbook.json (hunk 4 above)
Comment (aymenim) [reply Workbooks/BETTER_MTD_Workbook.json:1]: Included it in the BETTER MTD workbook to WorkbooksMetadata.json and added the required logo and preview images

File: DataConnectors/BETTERMTD.json [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+        {
+            "name": "BetterMTDNetflowLog_CL",
+            "lastDataReceivedQuery": "BetterMTDNetflowLog_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "BetterMTDIncidentLog_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
```
Comment (nazang) [inline DataConnectors/BETTERMTD.json:68]: The data connector will show as connected only if the specific table has logs, else even if other tables have logs and this doesn’t, (can happen if a certain type of logs are not there) it’ll show as disconnected. To fix this, multiple data types needs to be included as comma separated values in the connectivity query (it resolves as OR internally). We have a similar pattern for Carbon Black data connector (pasted below for quick reference).
           "value": [
	                "CarbonBlackEvents_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)",
	                "CarbonBlackNotifications_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)",
	                "CarbonBlackAuditLogs_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
	            ]

**Review by nazang — CHANGES_REQUESTED** [review]

File: DataConnectors/BETTERMTD.json (hunk 5 above)
Comment (nazang) [reply DataConnectors/BETTERMTD.json:68]: aymenim, thank you for the quick update. All other changes in this PR looks good.

Comment (aymenim) [reply DataConnectors/BETTERMTD.json]: Thank you

**Review by nazang — APPROVED** [review]

**Comment by nazang** [comment]
Approve

**Comment by nazang** [comment]
@aymenim, there is a conflict must be resolved. Could you resolve it?

File: Workbooks/WorkbooksMetadata.json [hunk 6]
```diff
@@ -1023,5 +1036,6 @@
     "templateRelativePath": "DataCollectionHealthMonitoring.json",
     "subtitle": "",
     "provider": "Microsoft"
+>>>>>>> upstream/master
```
Comment (nazang) [inline Workbooks/WorkbooksMetadata.json]: Seems from the merge conflict. Need to remove.

**Review by nazang — CHANGES_REQUESTED** [review]

**Review by nazang — APPROVED** [review]

**Pull request merged**